webclient/components/pt-info/single/1time-Mrow-mField/diagnosis/db/client-side/structure/patient-of-diagnosis-table.js

```diff
@@ -0,0 +1,35 @@
+// For docs read webclient/docs/models.md
+import clientSideTableManage from '~/components/core/crud/manage-rows-of-table-in-client-side-orm.js'
+
+import clientSideTblMasterDiagnosis from './master-of-diagnosis-table'
+const { v1: uuidv1 } = require('uuid')
+
+let count = 0
+const intUniqueID = () => ++count
+
+export default class clientSideTblPatientDiagnosis extends clientSideTableManage {
+  static entity = 'tablePatientOfDiagnosis'
+
+  static primaryKey = 'clientSideUniqRowId'
+
+  static fields() {
+    return {
+      ...super.fields(),
+
+      clientSideUniqRowId: this.uid(() => intUniqueID()),
+      masterDiagnosisId: this.uid(() => uuidv1()), // This is service statement ID assigned to this patient coming from master table
```

Comment:
Value of masterDiagnosisId should be 'this.number(null)' and this is assigned to this patient coming from master table